### DISCUSSION TITLE
Serial port on EKCB07CV3 control box and EBLQ07CV3 heat pump

### DISCUSSION THREAD

**Discussion by Samuel-SC** [raomin/ESPAltherma#251] [Q&A]
Hello everyone

I have the Daikin Altherma Monobloc EBLQ07CV3 heat pump and the EKCB07CV3 control box. The X10A port on the control box does not appear to be the serial port. Could someone tell me which port I have to use to connect the esp32 and the definition of the heat pump that I have to uncomment?
Thank you so much.
Best regards.

![IMG_20230128_211019](https://user-images.githubusercontent.com/89522694/219957370-e5d6a2e3-dcd5-4dbf-8b44-7ebfff1d3332.jpg)

![IMG_20230128_211028](https://user-images.githubusercontent.com/89522694/219957318-4080fb27-0c21-4b73-ad95-ffdd3442ad57.jpg)

**Comment by bashers222** [comment]
I have exactly the same unit.
The manual states the X12A is the equivalent port in the control box which ive confirmed
